client/a8c-for-agencies/components/form/field/index.tsx

```diff
@@ -8,6 +8,7 @@ type Props = {
 	sub?: string;
 	description?: string;
 	showOptionalLabel?: boolean;
+	validationText?: string;
```

Comment:
Based on the intention of the highlighting, I think maybe we can name this as `errorText` or `error` to emphasize that this is a negative effect.